adal/tests/Test.ADAL.NET.UIAutomation/ADALMobileTestHelper.cs

```diff
@@ -64,25 +64,25 @@ public void AcquireTokenSilentTestHelper(ITestController controller, UserQueryPa
 
             //Enter 2nd Resource
             controller.EnterText(CoreUiTestConstants.ResourceEntryID, CoreUiTestConstants.Exchange, false);
-            controller.DismissKeyboard();
 
             //Acquire token silently
             controller.Tap(CoreUiTestConstants.AcquireTokenSilentID);
 
             CoreMobileTestHelper.VerifyResult(controller);
         }
 
-        public void AcquireTokenInteractiveHelper(ITestController controller, UserQueryParameters userParams)
+        public IUser AcquireTokenInteractiveHelper(ITestController controller, UserQueryParameters userParams)
```

Comment:
>AcquireTokenInteractiveHelper [](start = 21, length = 29)

There is an architectural flaw with our tests, which you may or may not want to fix as part of this PR. To fetch a user you need a test controller, and we bundle it with pressing "clear cache" etc. This violates SRP, makes it difficult to debug failing tests etc. The test structure should be more in the lines of: 

1. Get a user from the lab. Optionally fetch its password too. Fail fast if you can't. 
2. Do the test (start pressing buttons etc.)

Reply:
Anyone can make this change while updating test code :)

Reply:
I made the changes. please see latest commit.